### DISCUSSION TITLE
Großschreibung in der Email-Adresse ist nicht erlaubt

### DISCUSSION THREAD

**Discussion by ElbPsi** [fabiankachlock/zwoo#257] [Bug Reports]
Hallo Team,
schade, dass nicht erklärt wird, warum die Email ungültig ist.
Grüße, Elb

![IMG-20220918-WA0000](https://user-images.githubusercontent.com/54574558/190911978-0bfb4681-bf04-4f0d-af00-39730797e7d8.jpg)

**Comment by fabiankachlock** [comment]
Hallo Elb,
vielen Dank für den Bug Report.

Zurzeit werden nur Kleinbuchstaben bei der Validierung der E-Mail beachtet. Da bei E-Mail Adressen Groß/Kleinschreibung i. d. R. nicht beachtet wird, sind hoffentlich keine weiteren Probleme aufgetreten. Trotzdem werden wir so schnell wie möglich diesen Fehler beheben.

zwoo Team

**Reply by fabiankachlock** [reply]
linked: #266; ZWOO-88

**Comment by fabiankachlock** [comment]
Lösungsversion: v1.0.0-beta.4
